{"ast":null,"code":"import{createErrorClass}from'./createErrorClass';export var SequenceError=createErrorClass(function(_super){return function SequenceErrorImpl(message){_super(this);this.name='SequenceError';this.message=message;};});","map":{"version":3,"names":["createErrorClass","SequenceError","_super","SequenceErrorImpl","message","name"],"sources":["/Users/jawaadahmar/Desktop/atlas-clothing/client/src/thistle-lion/node_modules/rxjs/src/internal/util/SequenceError.ts"],"sourcesContent":["import { createErrorClass } from './createErrorClass';\n\nexport interface SequenceError extends Error {}\n\nexport interface SequenceErrorCtor {\n  /**\n   * @deprecated Internal implementation detail. Do not construct error instances.\n   * Cannot be tagged as internal: https://github.com/ReactiveX/rxjs/issues/6269\n   */\n  new (message: string): SequenceError;\n}\n\n/**\n * An error thrown when something is wrong with the sequence of\n * values arriving on the observable.\n *\n * @see {@link operators/single}\n *\n * @class SequenceError\n */\nexport const SequenceError: SequenceErrorCtor = createErrorClass(\n  (_super) =>\n    function SequenceErrorImpl(this: any, message: string) {\n      _super(this);\n      this.name = 'SequenceError';\n      this.message = message;\n    }\n);\n"],"mappings":"AAAA,OAASA,gBAAgB,KAAQ,oBAAoB,CAoBrD,MAAO,IAAM,CAAAC,aAAa,CAAsBD,gBAAgB,CAC9D,SAACE,MAAM,EACL,eAAS,CAAAC,iBAAiBA,CAAYC,OAAe,EACnDF,MAAM,CAAC,IAAI,CAAC,CACZ,IAAI,CAACG,IAAI,CAAG,eAAe,CAC3B,IAAI,CAACD,OAAO,CAAGA,OAAO,CACxB,CAAC,CAJD,CAIC,CACJ"},"metadata":{},"sourceType":"module","externalDependencies":[]}